tests/ignite/contrib/handlers/test_tqdm_logger.py

```diff
@@ -365,8 +365,7 @@ def test_pbar_wrong_events_order():
     with pytest.raises(ValueError, match="should be called before closing event"):
         pbar.attach(engine, event_name=Events.ITERATION_COMPLETED, closing_event_name=Events.ITERATION_STARTED)
 
-    with pytest.raises(ValueError, match="Closing event should not use any event filter"):
-        pbar.attach(engine, event_name=Events.ITERATION_STARTED, closing_event_name=Events.EPOCH_COMPLETED(every=10))
+    pbar.attach(engine, event_name=Events.ITERATION_STARTED, closing_event_name=Events.EPOCH_COMPLETED(every=10))
```

Comment:
I'm not sure whether this works correctly...
As commented previously, we should not set closing events as filtered events.